source/help/messaging/attaching-files.rst

```diff
@@ -1,60 +1,56 @@
 Sharing Files
 ===============
 
-With attachments, you can share additional information that will help your 
-team to visually understand your ideas. Sharing videos, voice recordings,
-screenshots, and photos will make your messages more effective and clear.
+With file attachments, you can share additional information that will help your team to visually understand your ideas. Sharing videos, voice recordings, screenshots, and photos will make your messages more effective and clear.
 
-How to Attach Files
--------------------
-You can attach files in the following ways:
+Attach Files to Messages
+------------------------
 
--  Use the attachment icon - click the paperclip icon inside the message input box
+You can attach files to messages in the following ways:
+
+-  Use the attachment icon - select the paperclip icon inside the message input box
 -  Drag and Drop
 -  Paste from the clipboard
 
-File Previewer
---------------
+Preview File Attachments
+------------------------
+
 Mattermost has a built-in file previewer that you can use to:
 
 -  Download files
 -  Share public links
 -  View media
 
-Click the thumbnail of an attached file to open it in the file previewer.
+Select the thumbnail of an attached file to open it in the file previewer.
 
 Downloading Files
 ~~~~~~~~~~~~~~~~~
-You can download an attached file by clicking the download icon next to the file 
-thumbnail.
+
+You can download an attached file by selecting the download icon next to the file thumbnail.
 
 Sharing Public Links
 ~~~~~~~~~~~~~~~~~~~~
-Public URLs allow you to share attachments with anyone outside the Mattermost 
-system. To share an attachment, click the thumbnail of an attachment, then click
-**Get Public Link**.
 
-.. Tip::
-  If **Get Public Link** is not visible in the file previewer
-  and you prefer that the feature is enabled, you can request your System
-  Admin to enable the feature from the System Console under
-  **Security > Public Links**.
+Public URLs allow you to share attachments with anyone outside the Mattermost system. To share an attachment, select the thumbnail of an attachment, then select **Get Public Link**.
+
+.. tip::
```

Comment:
I changed the formatting of this slightly; will need to confirm that it renders as expected in live.